aspnet/publishing/iis.rst

```diff
@@ -1,116 +1,178 @@
 
-.. image:: pubIIS/_static/role-services.png
+  Keep in mind that if you use the **File Name Extensions** approach that your application will not receive requests nor be able to serve any file with that extension, even static files in your **webroot** that you wish to serve (e.g., JSON or XML files you have placed in `wwwroot` for legitimate file serving). Therefore, specifically naming sensitive files using the **Hidden Segments** approach might be preferable in most scenarios [i.e., explicitly naming **my_application.runtimeconfig.json**, **my_application.xml** (if any XML Documentation comments files are present), **my_application.deps.json**, and any other files that are not explicitly excluded by IIS for static file serving].
 
-.. _unlock-handlers:
+Configure the Website in IIS
+----------------------------
 
-3. Unlock the configuration section.
+#. In **IIS Manager**, create a new website. Provide a **Site name** and set the **Physical path** to the application's assets folder that you created. Provide the **Binding** configuration and create the website.
+#. Set the application pool to **No Managed Code**. ASP.NET Core runs in a separate process and manages the runtime.
 
-  - Launch IIS Manager and select the server in the **Connections** pane on the left (see image below).
-  - Double-click **Configuration Editor**.
-  - In the **Section** drop-down, select **system.webServer/handlers**, and then click **Unlock Section**.
+.. note:: If you change the default identity of the application pool from **ApplicationPoolIdentity** to some other user account, make sure you configure the correct permissions for the user identity to use the application's assets folder.
 
-.. image:: pubIIS/_static/config-edit.png
-.. image:: pubIIS/_static/unlock.png
+Open the **Add Website** window.
 
-- Set the application pool to **No Managed Code**. ASP.NET Core runs in a separate process and manages the runtime.
+	.. image:: pubIIS/_static/addwebsitecontextmenu.png
+	
+Configure the website.
 
- .. image:: pubIIS/_static/appPool.PNG
+	.. image:: pubIIS/_static/addwebsite.png
+	
+In the **Application Pools** panel, open the **Edit Application Pool** window by right-clicking on the website's application pool and selecting **Basic Settings...** from the popup menu.
 
+	.. image:: pubIIS/_static/basicsettingscontextmenu.png
+	
+Set the **.NET CLR version** to **No Managed Code**.
 
-Supported operating systems
------------------------------------
+	.. image:: pubIIS/_static/editapppool.png
```

Comment:
@shirhatti don't we want to show browse web site (show them how to browse site) at this point?